llvm/lib/SYCLLowerIR/LowerESIMD.cpp

```diff
@@ -1040,6 +1049,8 @@ static void translateESIMDIntrinsicCall(CallInst &CI) {
     return;
 
   auto *FTy = CI.getCalledFunction()->getFunctionType();
+  if (!FTy)
+    return;
```

Comment:
I assume that `CI.getCalledFunction()` result must be validated instead (or in addition to).

Reply:
fixed.